### DISCUSSION TITLE
How to change test name in allure-playwright? no "@DisplayName" annotation for allure-js

### DISCUSSION THREAD

**Discussion by tepmehatop** [allure-framework/allure2#2625] [Questions & Support] [answered]
Hi.
using allure-js (allure-playwright)
https://github.com/allure-framework/allure-js/blob/main/packages/allure-playwright/README.md

There is a need to change test name during the process.
As I understand it, in the Java version it is possible to specify a @DisplayName annotation, but how can this be done in the JS version?

Thanks in advance!

**Comment by baev** [comment, marked as answer]
Since 3.0.0 (currently in beta), you can use `displayName` runtime function:

```ts
import { test } from "@playwright/test";
import { displayName } from "allure-js-commons";

test("first test",  async() => {
  await displayName("changed name");
});
```
